pyproject.toml

```diff
@@ -20,3 +20,12 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 
+[tool.pytest.ini_options]
+addopts = ["--strict-config", "--strict-markers", "--flake8"]
+xfail_strict = true
+junit_family = "xunit2"
+filterwarnings = [
+    "error",
+    '''ignore:Text file object support is deprecated in favor of binary file objects\. Use `open\("foo.toml", "rb"\)` to open the file in binary mode\.:DeprecationWarning''',
```

Comment:
this deprecation warning is causing downstream failures:
https://github.com/pypa/setuptools/pull/2746/checks?check_run_id=3200086380#step:5:339